sample.config.js

```diff
@@ -45,4 +45,9 @@ module.exports = {
 		// 	certPath: '',
 		// },
 	},
+
+	// Third party APIs for command functionality
+	thirdParty: {
+		exchangeRates: '',
```

Comment:
Since this is the sample config, let's add a comment that clarifies what site someone setting the bot up needs to get a key from
```suggestion
		// https://exchangeratesapi.io
		exchangeRates: '',
```